<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>854</width>
      <height>900</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_defalut">
     <widget class="QLabel" name="label_info">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>751</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>初始界面，按左右键切换</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_info_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>751</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>初始界面，按左右键切换</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_option"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
